src/domain/synthetics/markets/utils.ts

```diff
@@ -20,19 +21,31 @@ export function getMarketFullName(p: { longToken: Token; shortToken: Token; inde
 }
 
 export function getGlvMarketName(chainId: number, address: string) {
-  return GLV_MARKETS_APPEARANCE[chainId]?.[address]?.name;
+  return GLV_MARKETS[chainId]?.[address]?.name;
 }
 
 export function getGlvMarketDisplayName(glv: GlvMarketInfo) {
```

Comment:
since we aim to not confuse glv and market in this PR, I'll ask to rename entities from prev iteration

I think this one should be getGlvDisplayName